cypresse2e/cypress/integration/e2eTests/post/post.spec.js

```diff
@@ -0,0 +1,75 @@
+import HelpBoard from '../../../elements/pages/helpBoard'
+import Post from '../../../elements/pages/post'
+
+describe('FightPandemics Post for unauthorized user', () => {
+
+    const helpBoard = new HelpBoard();
+    const post = new Post();
+    const shareViaModalWindowTitle = "Share via..."
+    var postTitle = '';
+
+    context('User opens Help Board', () => {
+        beforeEach(() => {
+            helpBoard.visit();
+        });
+
+        it('Unauthorized user is redirected to SignIn page when clicking on Like button', () => {        
+            cy.get(helpBoard.getFirstPostOnHelpBoardSelector()).within(($firstPost) => { 
+                cy.get(post.getLikeButtonSelector()).click();
+                cy.validateCorrectScreenIsOpen("auth/login");
+            })                                                      
+        });
+
+        it('Unauthorized user is redirected to SignIn page when clicking on Comment button', () => {     
+            cy.get(helpBoard.getFirstPostOnHelpBoardSelector()).within(($firstPost) => { 
+                cy.get(post.getCommentButtonSelector()).click();
+                cy.validateCorrectScreenIsOpen("auth/login");
+            })                                          
+        });
+
+        it('Unauthorized user can see a Share via... modal window', () => {  
+              
+            cy.get(helpBoard.getFirstPostOnHelpBoardSelector()).within(($firstPost) => { 
+                var postTitleElement = post.getPostTitle();
+                postTitleElement.invoke('text').then((text => {
+                    postTitle = text.toString();
+                }));
+                post.getShareButton().click();
+            })  
+            cy.get(post.getModalWindowShareViaSelector()).within(($modalWindow) => { 
+                var modalWindowTitle = post.getModalWindowShareViaH4Title();
+                modalWindowTitle.should('be.visible').contains(shareViaModalWindowTitle);
+                validateSocialMediaShareButton(post.getModalWindowEmailButton(), 'title', postTitle);
```

Comment:
Can _'aria-label', 'email'_ be used instead of _'title', postTitle_? Or does it cause an issue in the tests when using the aria-label? This would eliminate the need to to create the extra postTitle variable.

Reply:
Changed to aria-label